cloud-inquisitor/ledger/aws.go

```diff
@@ -0,0 +1,67 @@
+package ledger
+
+import (
+	"context"
+	"time"
+
+	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor"
+	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor/database"
+	"github.com/aws/aws-lambda-go/lambdacontext"
+	"github.com/jinzhu/gorm"
+	"github.com/mitchellh/mapstructure"
+)
+
+type AWSActionLedger struct {
```

Comment:
Can we include "entry" or "log" or something to denote that this is not the ledger object itself?